plugins/lookup/passwordstore.py

```diff
@@ -73,6 +77,12 @@
   vars:
     mypassword: "{{ lookup('community.general.passwordstore', 'example/test create=true')}}"
 
+- name: Outputs 'abc' without updating the the pass file if example/test doesn't exist, just give the password otherwise
```

Comment:
Is this going to update the pass file if it exists or is it just going to retrieve password?
```suggestion
- name: Prints 'abc' if example/test doesn't exist, just give the password otherwise
```